style/checkstyle.xml

```diff
@@ -46,10 +46,10 @@
         <module name="TypeName">
             <message key="name.invalidPattern" value="Type name ''{0}'' must match pattern ''{1}''."/>
         </module>
-        <module name="ClassTypeParameterName">
-            <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)"/>
-            <message key="name.invalidPattern" value="Class type name ''{0}'' must match pattern ''{1}''."/>
-        </module>
+<!--        <module name="ClassTypeParameterName">-->
+<!--            <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)"/>-->
+<!--            <message key="name.invalidPattern" value="Class type name ''{0}'' must match pattern ''{1}''."/>-->
+<!--        </module>-->
```

Comment:
could we consider use low limit check rather than disable all?

like `Class<a>` / `Class<1>` is not allowed

Reply:
I think we can implement and use the regular expression "[A-Z][a-zA-Z0-9]*$"

Reply:
> I think we can implement and use the regular expression "[A-Z][a-zA-Z0-9]*$"

Fine

Reply:
I had updated checkstyle file.